community/security/src/main/java/org/neo4j/server/security/auth/FileUserRepository.java

```diff
@@ -180,9 +180,15 @@ public int numberOfUsers()
     }
 
     @Override
-    public boolean isValidName( String name )
+    public boolean isValidUsername( String username )
     {
-        return name.matches( "^[a-zA-Z0-9_]+$" );
+        return username.matches( "^[a-zA-Z0-9_]+$" );
```

Comment:
And such a simple check can most likely be made faster with just character range checks, without must cost in complexity.